ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Block/SupportsPeras.hs

```diff
@@ -218,3 +394,51 @@ instance
   HasPerasCertBoost (WithArrivalTime cert)
   where
   getPerasCertBoost = getPerasCertBoost . forgetArrivalTime
+
+class HasPerasVoteRound vote where
```

Comment:
Do you think it would be a worthwhile idea to get rid of the triplicate boilerplate with a single `HasPerasVote` class? You would then get the 3 `getPerasVoteStuff` with `pvVoteStuff . getPerasVote`.